gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ObservationMapper.java

```diff
@@ -93,45 +98,75 @@ public List<Observation> mapResources(RCMRMT030101UKEhrExtract ehrExtract, Patie
     }
 
     private Observation mapObservation(RCMRMT030101UKEhrComposition ehrComposition,
-                                       RCMRMT030101UKObservationStatement observationStatement, Patient patient, List<Encounter> encounters,
+                                       RCMRMT030101UKObservationStatement observationStatement,
+                                       Patient patient,
+                                       List<Encounter> encounters,
                                        String practiseCode) {
 
+        var compoundStatement = ehrComposition
+            .getComponent()
+            .stream()
+            .flatMap(component4 -> extractAllCompoundStatements(component4))
```

Comment:
Can we substitute this with a method reference? E.g. `CompoundStatementResourceExtractors::extractAllCompoundStatements`?

Reply:
well-spotted 👍, will be fixed